plugins/chat/app/controllers/chat/api/tracking_controller.rb

```diff
@@ -0,0 +1,37 @@
+# frozen_string_literal: true
+
+class Chat::Api::TrackingController < Chat::ApiController
```

Comment:
I would suggest this as route to update last read:

```
PUT /channels/:id/read/:message_id
```

So we would have `Chat::Api::ChannelReadsController#update`